code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -111,6 +111,8 @@
 			S.loc = remote_eye.loc
 			S.visible_message("[S] warps in!")
 			X.stored_slimes -= S
+	else
+		owner <<"<span class='notice'>Target is not near a camera.  Cannot proceed.</span>"
```

Comment:
Two little nitpicks: you need to add a space between << and the message. Also, you put two spaces after the first period instead of one.
Same thing for the other three new lines below.